src/middleware/permissionsMiddleware.js

```diff
@@ -1,16 +1,36 @@
 import { rule, shield, and, or, not, allow } from 'graphql-shield'
 
 const isAuthenticated = rule()(async (parent, args, ctx, info) => {
+  // TODO: how to get this working while seeding?
+  console.log('isSeeding', process.env.IS_SEEDING)
+  if (process.env.IS_SEEDING === true) {
+    return true
+  }
   return ctx.user !== null
 })
 const isOwner = rule()(async (parent, args, ctx, info) => {
+  // TODO: how to get this working while seeding?
+  console.log('isSeeding', process.env.IS_SEEDING)
+  if (process.env.IS_SEEDING === true) {
+    return true
+  }
   console.log('parent', parent)
   return ctx.user.id === parent.id
 })
 const isAdmin = rule()(async (parent, args, ctx, info) => {
+  // TODO: how to get this working while seeding?
+  console.log('isSeeding', process.env.IS_SEEDING)
+  if (process.env.IS_SEEDING === true) {
```

Comment:
`isAuthenticated` and `isAdmin` should be the only required hooks. Why do we need `isModerator` and `isOwner`?

Reply:
because moderator can moderate other content and owners can edit its own data.